process/incident_management.md

```diff
@@ -0,0 +1,263 @@
+
+There can be multiple levels of severity for an incident depending on the impact it has. The process below would be used when the incident is considered to be a major incident. When the incident is a minor one then a lightweight ad-hoc process can be used. The goal being to make sure not to make the process too heavy for small incidents.
+
+A major outage can be defined when our platform is running out of error budget or when we start failing our SLOs.
+
+Examples of major incidents:
+* Full outage or heavy degradation of one or multiple services impacting one or several customers
+* Duration of the incident is greater than 4 hours
+* Multiple layers and SRE teams are impacted (platform, application, …)
+
+Note that when a team has a path for fast recovery then they shouldn't break it to fit that process if it's not required.
+
+## Quick reference guide
+In case of a customer–impacting incident (persisting or intermittent degradation of customer–exposed functionality):
+
+1. The Incident First Responder is the first SRE being alerted of the issue and starting the investigation
+2. After initial investigation the First Responder involves the Incident Commander (by default a Region Lead but could be Team Lead or Manager) to help with the coordination of the recovery efforts
+    1. First Responder opens the bridge call for that
+3. The Incident Commander manages the communication and coordination around the recovery
+    1. By creating a dedicated IM channel to record the audit trail of the incident resolution.
+    2. Post a notification in the relevant channels for external awareness.
+    3. By involving the right people to help and speed up the recovery process
+    4. By ensuring a good communication with first level support and the customer
+4. The incident is recovered
+5. The Incident Owner (usually the First Responder unless the Incident Commander picked someone else) is responsible for running and documenting the post mortem and root cause analysis and bring the incident to full closure.
+
+## Incidents
+An incident is an event that can occur at any time and which results in a degradation or outage of one or multiple of our services.
+
+An incident can be raised by a customer or directly by our monitoring and alerting systems, or directly by a member of the SRE team.
+
+Depending on the impact of the incident on the services and customers it will be categorized in terms of severity.
+
+As an SRE organization we should aim at:
+
+* Minimizing the time it takes us to recover the service (MTTR)
+* Improve Reliability of our services: i.e. having as few incidents as possible, optimizing prevention of incidents
```

Comment:
```suggestion
* Improve Reliability of our services: i.e.,  optimizing prevention of incidents
```
I don't believe that "having as few incidents as possible" is something actionable, it can happen anytime.

Reply:
it could be rephrased by saying that we aim at lowering the impact of any incident happening